addons/essentials/package.json

```diff
@@ -43,13 +43,15 @@
     "@storybook/addon-backgrounds": "6.3.0-beta.5",
     "@storybook/addon-controls": "6.3.0-beta.5",
     "@storybook/addon-docs": "6.3.0-beta.5",
+    "@storybook/addon-measure": "latest",
```

Comment:
This should be a `^x.x.x` version so you can release breaking changes to the addon without breaking `addon-essentials` users